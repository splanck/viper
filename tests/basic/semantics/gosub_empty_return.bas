10 GOSUB 100
20 RETURN
30 END
100 RETURN
