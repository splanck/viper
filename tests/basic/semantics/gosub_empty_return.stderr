Trap @main#1 line 2: DomainError (code=0): gosub: empty return stack
